osu.Game.Rulesets.Osu/Mods/OsuModRelax.cs

```diff
@@ -135,5 +141,9 @@ void changeState(bool down)
                 state?.Apply(osuInputManager.CurrentState, osuInputManager);
             }
         }
+
+        public void ApplyToScoreProcessor(ScoreProcessor scoreProcessor) => scoreProcessor.Combo.UnbindEvents();
```

Comment:
i'm not sure what the intention behind this is, but this is *very* wrong, and *will* break scoring algorithms. actually tempted to go and PR a change of `Combo` to `IBindable` so that you can't do this.

if this is intended to hide the HUD/combo counter from the player, you're looking for `IApplicableToHUD`.

Reply:
It's intended to affect scoring and combo hud should still be visible